### DISCUSSION TITLE
Ability To Remove "Recently Watched" From Sidebar

### DISCUSSION THREAD

**Discussion by JWParenteau** [night/betterttv#6036] [Ideas]
Not sure if it's a new Twitch thing, although, there seems to be a new side bar called "Recently Watched" the information icon states when hovering over:
"Live channels that you've watched in the last 30 days that you didn't follow."

I'd like to be able to pull/hide this section.

**Comment by night** [comment]
This is now supported via Settings → Sidebar → toggle off Recently Watched Channels. Closing as resolved.
